willYouSleep2.0.js

```diff
@@ -0,0 +1,320 @@
+const readline1 = require("readline-sync");
+console.clear();
+
+let nameInput = readline1.question("Enter your name: ");
+console.clear();
+
+console.log(`Hello ${nameInput}!  Welcome to Will you sleep? `);
+
+console.log(
+  "\nTo play this game, you will be given different choices on whether you want to remain asleep or stay awake! \n"
+);
+
+const ageRequirement = () => {
+  let age = readline1.question("How old are you?\n");
+  if (age < 18) {
+    console.log("You are a minor");
+    endGame()
+  } else if (age === 18) {
+    console.log(`${age}, finally! Congratulations on being an adult!`);
+    startGame()
+  } else {
+    console.log(`You're old enough to play!\n`);
+    startGame()
+  }
+};
+
+function decisions() {
+  console.log("\nIts 5am and a explosion wakes you up!\n");
+  let person = ["Roll over", "Go check it out"];
+  let index = readline1.keyInSelect(person, "Do you?");
+  switch (person[index]) {
+    case person[0]:
+      rollOver();
+      break;
+    case person[1]:
+      checkItOut();
+      break;
+    case person[-1]:
+        startGame()
+        break; 
+  }
+}
+
+function checkItOut() {
+  console.clear();
+  console.log("Let's go check! should we...\n");
+let person = ['Get dressed! and see if neighbors are up?','Look out the window?', 'Go back to sleep?"']
+let index = readline1.keyInSelect(person, 'Do you?')
+  if (person[0]) {
+    getDressed();
+  } else if (person[1]) {
+    theWindow();
+  } else if (person[2]) {
+    whoAmIKidding();
+  } else {
+      startGame
+  }
+
+}
+
+function rollOver() {
+  console.clear();
+  person = readline1.question("Good job! sleeping is IMPORTANT!! ");
+  startGame();
+}
+
+function whoAmIKidding() {
+  console.clear();
+  console.log(
+    "Who am I kidding! I always knew it would end somehow/someday! \n"
+  );
+  startGame();
+}
+
+function getDressed() {
+  console.clear();
+  console.log(
+    "You get dressed and go out into the hallway to find out where the explosion sound came from\n"
+  );
+  console.log("You see everyone's confused in the hallway in their pajamas \n");
+
+  let person = readline1.question(
+    "A: Do you ask clueless neighbors,    B: Follow dog,    C: Put on your pjs back on \n"
+  );
+  person = person.toUpperCase()
+  if (person === "A") {
+    askNeighbors();
+  } else if (person === "B") {
+    followDog();
+  } else if (person === "C") {
+    putOnPjs();
+  } else {
+    startGame()
```

Comment:
You can let the user know that what they entered was an invalid option. You place this logic in a while loop. Loop while the `person` variable does not contain the string letters `A, B, or C`